src/pages/WebMeetWaitingRoom.js

```diff
@@ -244,7 +244,8 @@ const WebMeetWaitingRoom = ({ patientDetails, hashKey }) => {
 
   return (
     <>
-      {appointmentStartDate.date === today &&
+      {
+      // appointmentStartDate.date === today &&
```

Comment:
@Shreets please do not comment the code, disable the condition with another temp condition using config and adding toDo comment